app/views/shared/footer/_footer.html.erb

```diff
@@ -11,6 +11,9 @@
           <%= render 'shared/footer/footer_social' %>
           <%= render 'shared/footer/footer_newsletter' %>
         </div>
+        <div class="footer__copyright">
```

Comment:
This should be p tag not a div
you'll probably need to add margin: 0 the the above styles

Reply:
done, @jiveturkeyJay ♻️